src/docs/_sass/main.scss

```diff
@@ -1,5 +1,3 @@
----
----
```

Comment:
these dashes were telling jekyll to build this file, without them he won't touch it